docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -0,0 +1,59 @@
+
+## Configuring z/OS console REST interface
+
+Review the following recommendations for configuring the security for z/OS console REST services:
+
+- Add the COMMON_TSO statement to the IZUPRMxx parmlib member to customize the z/OSMF options for the logon procedure.
+- Define a value of at least 50000 KB as the size of the address space for the user's logon procedure. To help you prevent system memory exception errors from occurring, confirm that this value is acceptable in your environment.
+- Ensure that the members in your z/OSMF user security group can issue TSO and CONSOLE commands. IBM provides RACF statements for user group IZUUSER. To prevent all z/OSMF users from issuing TSO and CONSOLE commands, you can create more z/OSMF user groups for more granular security.
+- Ensure that the OPERCMD class is active and that your MVS commands are protected. MVS commands include, but are not limited to, the MVS and MVS.MCSOPER resource prefixes.
+- Ensure that the z/OSMF user security groups can access (authorized) the logon procedure name and account number that is specified in the COMMON_TSO statement.
+- Define a TSO segment for all the z/OSMF users.
+
+## Configuring z/OS data set and file REST interface
+
+Review the following recommendations for configuring the z/OS security for data set and file REST services:
+
+- Add the COMMON_TSO statement to the RESTAPI_FILE parmlib member to customize the z/OSMF options for the logon procedure.
+- Define a value of at least 65536 KB as the size of the address space for the user's logon procedure. To help you prevent system memory exception errors from occurring, confirm that this value is acceptable in your environment.
+- Authorize z/OSMF user groups and the z/OSMF server to CEA TSO/E address space services.
+- Ensure that the z/OSMF user security groups can access (authorized) the logon procedure name and account number that is specified on the COMMON_TSO statement.
+- Define a TSO segment for all the z/OSMF users.
+- Define at least 20971520 KB (20 MB) the IPCMSGQBYTES option of your parmlib member named BPXPRMxx. IBM recommends this value to let TSO and z/OSMF communicate using z/OS USS interprocess communications.
+
+## Configuring z/OSMF plug-in security
+Ensure that you implement all the required security for the plug-ins. For more information, see [Setting up security for the z/OSMF plug-ins](https://www.ibm.com/docs/en/zos/2.2.0?topic=configuration-setting-up-security-zosmf-plug-ins) in the IBM Documentation.
+
+:::note
+
+- For systems that are secured by RACF, ensure that the TRUSTED attribute is assigned to the CEA started task.
+- To implement the use of certificates to access Zowe CLI **[correct?]**, security administrators can configure the certificates for Zowe CLI **[correct?]** users. For more information, see [Using the z/OSMF REST services](https://www.ibm.com/docs/en/zos/2.2.0?topic=guide-using-zosmf-rest-services) in the IBM Documentation.
```

Comment:
@adam-wolfe @zFernand0: I took this content from the Brightside doc. No surprisingly, in the Brightside doc, "Zowe CLI" was actually "Brightside." In this file, I replaced all mentions of "Brightside" with "Zowe CLI." But I'd like to confirm that the substitutions are one-for-one. 
Is this reference to "Zowe CLI" correct here?

Reply:
I would say "To implement the use of certificates to access **z/OSMF**, ..." or "To enable Zowe CLI to authenticate to z/OSMF using certificates, ...".

The second instance of Zowe CLI is correct here.